Client/Frontend/Browser/Onboarding/OnboardingViewController.swift

```diff
@@ -5,21 +5,37 @@
 import UIKit
 import BraveShared
 import Shared
+import BraveRewards
 
 /// A base class to provide common implementations needed for user onboarding screens.
-class OnboardingViewController: UIViewController {
+class OnboardingViewController: UIViewController, Themeable {
     weak var delegate: Onboardable?
     var profile: Profile
-    let theme: Theme
+    var rewards: BraveRewards?
+    var theme: Theme
     
     /// Whether the on-boarding is dark or not, based solely on passed in theme
     ///  Added explicitly to make removal easier in the future if just `theme` is used
     var dark: Bool {
         return theme.isDark
     }
     
-    init(profile: Profile, theme: Theme) {
+    static func colorForTheme(_ theme: Theme) -> UIColor {
+        return theme.isDark ? UIColor(rgb: 0x343A40) : UIColor(rgb: 0xFFFFFF)
+    }
+    
+    static func getUpdatedTheme() -> Theme {
+        let fallback = Theme.DefaultTheme.light.rawValue
+        if #available(iOS 13.0, *) {
+            let isDark = UITraitCollection.current.userInterfaceStyle == .dark
+            return Theme.from(id: isDark ? Theme.DefaultTheme.dark.rawValue : fallback)
+        }
+        return Theme.from(id: fallback)
+    }
```

Comment:
@iccub @jhreis Same here

Reply:
This code is copy pasted into onboarding from 

```
/// Returns the theme of the given Tab, if the tab is nil returns a regular theme.
    ///
    /// - parameter tab: An object representing a Tab.
    /// - returns: A Tab theme.
    static func of(_ tab: Tab?) -> Theme {
```

in order to get the most recent theme info.. This is done because `Theme` is a struct (copy) and does NOT update automatically (class reference) if the user goes and switches the system to light/dark mode.